oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/IDManager.java

```diff
@@ -268,6 +268,32 @@ public static String buildId(String instanceId, String name) {
             return Hashing.sha256().newHasher().putString(String.format("%s_%s",
                     name, instanceId), Charsets.UTF_8).hash().toString();
         }
+
+        /**
+         * @return encoded process relation id
+         */
+        public static String buildRelationId(ProcessRelationDefine define) {
+            return define.sourceId + Const.RELATION_ID_CONNECTOR + define.destId;
```

Comment:
Is the length of process relation ID enough for source and dest IDs? What is the max length of process ID?

Reply:
The string fixed length of a single process ID is `64`. The length of two process IDs + `Const.RELATION_ID_CONNECTOR` is `129`, so it's ok to be the entity ID.